jobboard/migrations/0007_add_example_companies.py

```diff
@@ -0,0 +1,36 @@
+from django.db import migrations, transaction
+from recruiter.models import Company
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('jobboard', '0001_initial'),
+    ]
+
+    def generate_data(apps, schema_editor):
+
+        with transaction.atomic():
```

Comment:
The Company model belongs to the `recruiter` app, so why does this migration in the `jobboard` app? the end result may be that the migration order is messed up since we do not define the right dependencies.

Reply:
I do that because our other example companies also were there, so I think that that is their place. I will add the migrations over there, is that ok to delete these files in my local computer and add the new file on recruiter?  that will fix the right dependencies back?

Reply:
Save the code that you want to add somewhere on your computer,
then on your local branch, restore all the migration files to how they are on upstream/main.
Then create a new migration file in the correct location with the code you saved.

Reply:
hey, guy thx for your help! that's worked. 
I upload the updated version. Good night.